introduction

ashlarbox opened the pull request.

PeterBell committed:
Merge pull request #2 from githubteacher/master

Most Senseis should be respected

Commit:
Initial kill list

PeterBell committed:
Merge pull request #3 from deadlyvipers/kill_list

Initial kill list

Commit:
We dont need to kill Budd

ashlarbox committed:
introduction

codeschool-kiddo commented on introduction.md:
```diff
@@ -0,0 +1 @@
+Hello from Duke @ AshlarBox!
```
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

ashlarbox committed:
introduction with path comment

ashlarbox committed:
skills

ashlarbox committed:
skills

codeschool-kiddo committed:
Added new rule about coffee

ashlarbox committed:
rules

ashlarbox committed:
Merge pull request #2 from ashlarbox/new_rules

Too many people are spilling coffee

ashlarbox committed:
merging in deadly_skills

ashlarbox committed:
new kill list

ashlarbox committed:
merge v1.0.1

s/v1.0.1'

new kill list

# Please enter a commit message to explain why this merge is necessary,
# especially if it merges an updated upstream into a topic branch.
#
# Lines starting with '#' will be ignored, and an empty message aborts
# the commit.

ashlarbox committed:
new kill list

ashlarbox committed:
fixes #3

ashlarbox committed:
Merge pull request #4 from ashlarbox/kill_list

Kill list

ashlarbox committed:
Update README.md
